spring-integration-ftp/src/main/java/org/springframework/integration/ftp/dsl/FtpMessageHandlerSpec.java

```diff
@@ -30,7 +30,7 @@
  *
  * @author Artem Bilan
  * @author Joaquin Santana
- *
+ * @author Deepak Gunasekaran
  * @since 5.0
```

Comment:
There must be a blank line before `@since` on the class level JavaDocs. This way the author list is much visible.